src/main.cpp

```diff
@@ -1443,6 +1443,19 @@ bool CheckTransactionWithoutProofVerification(const CTransaction& tx, CValidatio
         }
     }
 
+    // Check for duplicate orchard nullifiers in this transaction
+    {
+        set<uint256> vOrchardNullifiers;
```

Comment:
Non-blocking (we'll fix all these in a backported refactor eventually):
```suggestion
        std::set<uint256> vOrchardNullifiers;
```